### PR TITLE
1. Reimplement oai mappers

### PR DESCRIPTION
**Pull request by lthurston**
Follows https://github.com/ucldc/rikolti/pull/312

### CODE REVIEW

File: metadata_mapper/mappers/mapper.py [hunk 1]
```diff
@@ -131,6 +130,19 @@ def collate_fields(self, fieldlist):
 
         return collated
 
+    def first_string_in_field(self, field):
+        """
+        TODO: get approval on method name
+
+        Fetches a field value from source_metadata, and returns it if it's a string, or the first
+        item if it's a list
+        """
```
Comment (lthurston) [inline metadata_mapper/mappers/mapper.py:139]: This needs an approved name, @amywieliczka

Comment (amywieliczka) [reply metadata_mapper/mappers/mapper.py:139]: !!! Approved 💯

**Review by amywieliczka — DISMISSED** [review]

**Comment by lthurston** [comment]
@amywieliczka This needs another approval because I removed the todo about method name approval. Going forward I'll leave such comments out of the code to avoid this.

**Review by amywieliczka — APPROVED** [review]

**Comment by amywieliczka** [comment]
@lthurston done!

**Pull request merged**